tests/test_wave_01.py

```diff
@@ -142,13 +139,9 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
+    assert movie_to_watch in updated_data["watched"]
```

Comment:
This is great! Notice that two more assertions could be added -  the one you have plus one for checking which movie is remaining int the watchlist list, and one for the other movie we expect to find in the watched list